### DISCUSSION TITLE
Delete Multiple Uploaded Files

### DISCUSSION THREAD

**Discussion by ChrisKatsaras** [strapi/strapi#6241] [Legacy Topics]
Currently, it's possible to delete multiple entries of a collection at once but as far as I can see you cannot do the same with file uploads. Maybe I'm missing something but it would be great to be to delete multiple files at once.
Any help from the community is greatly appreciated!

**Comment by lauriejim** [comment]
Hello! what is the version of Strapi you are using?
Can you also precise what you are trying to do because I'm not 100% sure to understand where you are in the application.

**Comment by ChrisKatsaras** [comment]
Hi @lauriejim ! I'm currently running version `3.0.0-beta.19.5`.
Sorry, I should have provided more detail. When you're viewing your content in Strapi, you have the option to multi-select and delete items as shown below: 

![image](https://user-images.githubusercontent.com/11506653/82719693-dd3e1d00-9c7a-11ea-8307-6d032c4c9a52.png)

However, when I attempted to do the same thing with `Files Upload` I couldn't seem to find the option to delete multiple files at once in the UI. Instead, the only option is to delete them one by one using the delete option on the far right:
![image](https://user-images.githubusercontent.com/11506653/82719717-15ddf680-9c7b-11ea-89c5-53f89a75ea7a.png)

There have been quite a few times where I've needed to delete multiple files and I'm not sure if the feature exists or if I'm simply missing it.
Any information you can give me is greatly appreciated!

**Reply by derrickmehaffy** [reply]
I believe this was added in the media library feature in beta.20.3

**Reply by ChrisKatsaras** [reply]
You're right! I updated Strapi and now I have the ability to delete multiple files at once! Thanks a lot @derrickmehaffy
